networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/gossip/forks/GossipForkManager.java

```diff
@@ -300,11 +302,9 @@ private boolean isActive(final GossipForkSubscriptions subscriptions) {
     return activeSubscriptions.contains(subscriptions);
   }
 
-  private void startSubscriptions(final GossipForkSubscriptions subscription) {
+  private synchronized void startSubscriptions(final GossipForkSubscriptions subscription) {
```

Comment:
all calls are from `configureGossipForEpoch` which is already synchronized and looks sync. Is synchronizatiion needed here too?